bin/oneoff/upload_non_en_videos.rb

```diff
@@ -0,0 +1,126 @@
+# This script is used to batch upload non-EN videos, with the assumption that
+# we will frequently get these in batches. They will be uploaded to YouTube
+# and S3 as well as added to videos.csv.
+
+require 'google/apis'
+require 'google/apis/youtube_v3'
+require 'google/api_client/client_secrets.rb'
+require 'googleauth'
+require 'googleauth/stores/file_token_store'
+require 'csv'
+require 'fileutils'
+gem 'google-api-client', '>0.7'
+require 'rubygems'
+require 'csv'
+
+require_relative '../../dashboard/config/environment'
+
+# The directory where the files are in. There can be multiple folders in this folder,
+# but those must be noted in the map file below
+VIDEO_FILE_DIRECTORY = ''
+# CSV that maps video keys to their title and filename
+# Must have columns titled 'Folder', 'Partner filename', 'Key', and 'Video Title'
+MAP_FILE = 'test.csv'
+# Switch this to false when ready to run
+DRY_RUN = true
+LOCALE = 'es-MX'
+
+APPLICATION_NAME = 'YouTube Data API Ruby Tests'
+
+# REPLACE WITH NAME/LOCATION OF YOUR client_secrets.json FILE
+CLIENT_SECRETS_PATH = ''
```

Comment:
If you wanted to add a `CDO.youtube_client_secret` to locals.yml and reference it here, I think you can do so by requiring `../deployment.rb`.

https://github.com/code-dot-org/code-dot-org/blob/2e347ef7fc53a3025d3bd364aa6ec7b3db61c250/deployment.rb#L45-L52

Reply:
Ahh it might already be present via `require_relative '../../dashboard/config/environment'`.